### PR TITLE
Doc && composer.json updated

### PR DESCRIPTION
**Pull request by ternel**
Just updated the doc to use composer (no more deps), and added the annotations versions for Tag and Tagging classes.
Composer.json updated, require symfony/framwork-bundle >=2.0 to work with sf2.1

### CODE REVIEW

**Comment by Nek-** [comment]
+1 !

**Comment by mweimerskirch** [comment]
+1

@ternel you could squash the 6 commit messages into one to make the pull request lighter:
git rebase -i HEAD~6
git push -f origin BRANCH_NAME

**Comment by ternel** [comment]
Thx for the tips :)

File: README.md [hunk 1]
```diff
@@ -16,30 +16,23 @@ library, which handles most of the hard work.
...
+    ```
 
-### Update the autoloader
+Or you can edit your composer.json, and add :
 
-Next, update your autoloader and add the following entry:
+    "require": {
+        "php": ">=5.3.2",
+        "symfony/symfony": "2.1.*",
+        "_comment":"your ther packages",
```
Comment (FabienPennequin) [inline README.md]: useless line

Comment (ternel) [inline README.md]: Deleted ;)

**Comment by FabienPennequin** [comment]
Thanks

**Pull request merged**